sev/src/firmware/linux/mod.rs

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod ioctl;
+
+#[derive(Copy, Clone, Debug, PartialEq)]
+#[repr(u8)]
+pub enum State {
+    Uninitialized,
+    Initialized,
+    Working,
+}
```

Comment:
Why not use the definition of this enum in `sev/src/firmware/mod.rs`?

Reply:
Sounds good, thanks for pointing out!